events/lastAvatarStandingTrivia/clientScripts/triviaMasterClientStandard.js

```diff
@@ -578,13 +586,13 @@
                     prizeMoney = MIN_PRIZE;
                 }
                 Entities.callEntityServerMethod(gameZoneProperties.id, "playSound", ['POT_DECREASE_SFX']);
-                Entities.callEntityServerMethod(gameZoneProperties.id, "loseCoins");
+                // Entities.callEntityServerMethod(gameZoneProperties.id, "loseCoins");
```

Comment:
commented out line

Reply:
This is still needed for trivia in the trivia domain, it was only removed for multi-con